Octokit.Tests.Integration/fixtures/OrganizationsHooksCollection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xunit;
+
+namespace Octokit.Tests.Integration.fixtures
```

Comment:
There are already 2 other existing classes in here with that lowercase name.  I agree it doesnt look nice but from a consistency point of view, this PR is consistent with all other occurences.  A separate issue/PR can be raised to tidy this up